obspy/clients/filesystem/tests/test_sds.py

```diff
@@ -222,6 +222,48 @@ def test_sds_report(self):
             if failed:
                 raise Exception
 
+    def test_get_waveforms_bulk(self):
+        """
+        Test get_waveforms_bulk method.
+        """
+        for year, doy in ((2015, 213), (2015, 1), (2012, 1)):
+            t = UTCDateTime("%d-%03dT00:00:00" % (year, doy))
+            with TemporarySDSDirectory(year=year, doy=doy) as temp_sds:
+                chunks = [
+                    ["AB", "XYZ", "", "HHZ", t,
+                    t + 20, os.path.join(temp_sds.tempdir, "file_1.mseed")],
+                    ["AB", "XYZ", "", "HHN", t + 20,
+                    t + 40, os.path.join(temp_sds.tempdir, "file_2.mseed")],
+                    ["AB", "XYZ", "", "HHE", t + 40,
+                    t + 60, os.path.join(temp_sds.tempdir, "file_3.mseed")],
+                    ["CD", "ZZZ3", "00", "BHZ", t + 60,
+                    t + 80, os.path.join(temp_sds.tempdir, "file_4.mseed")],
+                    ["CD", "ZZZ3", "00", "BHN", t + 80,
+                    t + 100, os.path.join(temp_sds.tempdir, "file_5.mseed")],
+                    ["CD", "ZZZ3", "00", "BHE", t + 120,
+                    t + 140, os.path.join(temp_sds.tempdir, "file_6.mseed")]
+                ]
+                client = Client(temp_sds.tempdir)
+                ret_val = client.get_waveforms_bulk(chunks)
```

Comment:
usually a variable like this will be called either just `st` to emphasize its type, or if you want to emphasize that it is what's coming out of a to-be-tested routine, call it `got` (we do this mostly when we compare it with an expected result variable named `expected`). this is just cosmetics, but consistency even if it is just naming variables, helps the code base a lot. :-)